app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    orig_file, new_file = command[3:].split(" ")
+    if orig_file == new_file:
+        return
```

Comment:
better to raise some Exception, so that the user will understand if something goes wrong